readthedocs/embed/views.py

```diff
@@ -218,13 +219,15 @@ def _get_doc_content(project, version, doc):
         include_file=False,
         version_type=version.type,
     )
-    storage_path += f'/{doc}.fjson'
-
-    if not build_media_storage.exists(storage_path):
-        log.info('File does not exist. file=%s', storage_path)
-        return
-
-    return json.load(build_media_storage.open(storage_path))
+    file_path = build_media_storage.join(storage_path, f'{doc}.fjson')
+    try:
+        with build_media_storage.open(file_path) as file:
+            file_contents = file.read()
+            return json.loads(file_contents)
```

Comment:
```suggestion
            return json.load(file)
```

There is no need to load the file to a string first and then convert it to dict. We can do it in one step.